boot/zephyr/serial_adapter.c

```diff
@@ -191,7 +191,7 @@ static int
 boot_uart_fifo_init(void)
 {
 #ifdef CONFIG_BOOT_SERIAL_UART
-	uart_dev = device_get_binding(CONFIG_UART_CONSOLE_ON_DEV_NAME);
+	uart_dev = device_get_binding(DT_UART_CONSOLE_ON_DEV_NAME);
```

Comment:
@ChristopherBittner @nvlsianpu This is probably still needed regardless.

Reply:
I opened PR